backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5683850_sys_gh14602_payment_term_adj.sql

```diff
@@ -0,0 +1,292 @@
+-- UI Element: Payment Term(141,D) -> Payment Term(184,D) -> main -> 20 -> flags.Nächster Geschäftstag
```

Comment:
I do not see the german translations. Neither for columns and neither for reference lists.

Reply:
![image](https://user-images.githubusercontent.com/438169/229634391-fb7960ca-8d20-4149-b411-cdde0ac77007.png)

Reply:
@chems, Add some german trl any way and we can revise later. But do not leave it empty!